rasa/core/actions/action.py

```diff
@@ -92,6 +97,8 @@ def action_from_name(
         return defaults[name]
     elif name.startswith(UTTER_PREFIX):
         return ActionUtterTemplate(name)
+    elif name.startswith(RESPOND_PREFIX):
```

Comment:
any way we could get away without adding this prefix but using the `utter` one?

Reply:
We wanted to have a new action type with a new prefix, since the behaviour of this action is very different from `UtterTemplate` action and also it helps in then distinguishing open domain response actions from all others.
Potentially in future, the feature could use a different NLG component too.